kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -147,6 +147,48 @@
         "description": "Network plugin which will deployed in Kubernetes cluster"
       },
       "type": "string"
+    },    
+    "windowsAdminUsername": {
+      "defaultValue": "azureuser",
+      "metadata": {
+        "description": "User name for the Windows Virtual Machines that are part of the Kubernetes cluster."
+      },
+      "type": "string"
+    },
+    "windowsAdminPassword": {
+      "defaultValue": "MN&d?2,@~j'c3;T+",
```

Comment:
I would not have a default value for password

Reply:
Agree. I forgot to remove. Updated the code.